e2e-with-targets/_targets.R

```diff
@@ -0,0 +1,62 @@
+# Created by use_targets().
+# Follow the comments below to fill in this target script.
+# Then follow the manual to check and run the pipeline:
+#   https://books.ropensci.org/targets/walkthrough.html#inspect-the-pipeline # nolint
+
+# Load packages required to define the pipeline:
+library(targets)
```

Comment:
It's kind of confusing that one time there's box and other times its `library` or `source`. Can't we just use `box` or global environment? Targets work fine if we use `box` in `_targets.R` script